README.md

```diff
@@ -89,6 +89,16 @@ Denylists can be manually placed in the `$RAINBOW_DATADIR/denylists` folder too.
 
 See [NoPFS](https://github.com/ipfs-shipyard/nopfs) for an explanation of the denylist format. Note that denylists should only be appended to while Rainbow is running. Editing differently, or adding new denylist files, should be done with Rainbow stopped.
 
+## Garbage Collection
+
+Over time, the flatfs datastore can fill up with previously fetched blocks. To free up this used disk space, garbage collection can be run. Garbage collection needs to be manually triggered. This process can also be automated by using a cron job.
```

Comment:
```suggestion
Over time, the datastore can fill up with previously fetched blocks. To free up this used disk space, garbage collection can be run. Garbage collection needs to be manually triggered. This process can also be automated by using a cron job.
```

Thanks for adding the docs here, just removed the flatfs mention because it doesn't really matter.